spyder/plugins/shortcuts/confpage.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Shortcut configuration page."""
+
+# Standard library imports
+import re
+
+# Third party imports
+from qtpy import PYQT5
+from qtpy.QtWidgets import (QHBoxLayout, QLabel, QMessageBox, QPushButton,
+                            QVBoxLayout)
+
+# Local imports
+from spyder.api.preferences import PluginConfigPage
+from spyder.api.translations import get_translation
+from spyder.plugins.shortcuts.widgets.table import (ShortcutFinder,
+                                                    ShortcutsTable)
+from spyder.utils import icon_manager as ima
+
+# Localization
+_ = get_translation('spyder')
+
+
+class ShortcutsConfigPage(PluginConfigPage):
+    APPLY_CONF_PAGE_SETTINGS = True
+
+    def setup_page(self):
+        # Widgets
+        self.table = ShortcutsTable(self, text_color=ima.MAIN_FG_COLOR)
+        self.finder = ShortcutFinder(self.table, self.table.set_regex)
+        self.label_finder = QLabel(_('Search: '))
+        self.reset_btn = QPushButton(_("Reset to default values"))
+        self.top_label = QLabel(
+            _("Here you can browse the list of all available shortcuts in "
+              "Spyder. You can also customize them by double-clicking on any "
+              "entry in this table."))
+
+        # Widget setup
+        self.table.finder = self.finder
+        self.table.set_shortcut_data(self.plugin.get_shortcut_data())
+        self.table.load_shortcuts()
+        self.table.finder.setPlaceholderText(
+            _("Search for a shortcut in the table above"))
+        self.top_label.setWordWrap(True)
+
+        # Layout
+        hlayout = QHBoxLayout()
+        vlayout = QVBoxLayout()
+        vlayout.addWidget(self.top_label)
+        hlayout.addWidget(self.label_finder)
+        hlayout.addWidget(self.finder)
+        vlayout.addWidget(self.table)
+        vlayout.addLayout(hlayout)
+        vlayout.addWidget(self.reset_btn)
+        self.setLayout(vlayout)
+
+        self.setTabOrder(self.table, self.finder)
+        self.setTabOrder(self.finder, self.reset_btn)
+
+        # Signals and slots
+        if PYQT5:

```

Comment:
There's no need to maintain this `if` anymore.

Reply:
Fixed.